src/factory.js

```diff
@@ -254,6 +254,30 @@ const createFSPIOPErrorFromJoiError = (error, cause, replyTo) => {
   return createFSPIOPError(fspiopError, msg, stackTrace, replyTo)
 }
 
+/**
+ * Factory method to create an FSPIOPError from a openapi-backend error.
+ *
+ * @param error {Error} - the openapi error
+ * @param replyTo {string} - the FSP to notify of the error if applicable
+ * @returns {FSPIOPError}
+ */
+const createFSPIOPErrorFromOpenapiError = (errors, replyTo) => {
```

Comment:
`errors` --> `error`

This function should always take a single error in, and rather leave it up to the application calling this function to handle the list of errors.